packages/build-utils/src/types.ts

```diff
@@ -312,13 +312,16 @@ export interface Builder {
   config?: Config;
 }
 
+export type LauncherType = 'Nodejs' | 'edge-light';
```

Comment:
This doesn't follow the convention of PascalCase. I'm thinking it should be `EdgeLight` but not sure.

@TooTallNate @EndangeredMassa Do you have a preference here?

Reply:
I'm out of the loop on Edge Light, but I agree that this value should be `EdgeLight` unless there are constraints from other parts of the system.

Reply:
Really not a strong opinion.
I meant to use the WinterCG key because when running this signature, we aim to provide our runtime (even if not using edge-runtime directly here).
In my first experiment, I used `Web` as a key, but it didn't felt good.

Happy to switch to `EdgeLight`, unless a better proposal arises.

Reply:
Matching the WinterCG key sounds reasonable. I'd rather a code search of `edge-light` show all references than maintain casing style in this type.